cmd/createtree/main_test.go

```diff
@@ -46,7 +58,7 @@ func TestRun(t *testing.T) {
 		HashStrategy:       trillian.HashStrategy_RFC_6962,
 		HashAlgorithm:      sigpb.DigitallySigned_SHA256,
 		SignatureAlgorithm: sigpb.DigitallySigned_RSA,
-		PrivateKey:         anyKey,
+		PrivateKey:         marshalAny(&keyspb.PrivateKey{Der: pemDer}),
```

Comment:
`marshalAny` is used twice.  I could remove it and do all of the marshaling before the test to take advantage of `t.Fatalf`, but my thought was that the values we are marshaling are test configuration values which won't change as a result of code, and that it was nice to be able to use the output of `marshalAny` directly in building the table driven tests.